### DISCUSSION TITLE
TS global.d.ts定义的全局变量访问报错

### DISCUSSION THREAD

**Discussion by lcx147258369** [NervJS/taro#12444] [Q & A]
### 相关平台
微信小程序

### 复现仓库
[https://github.com/lcx147258369/taro-demo-test](https://github.com/lcx147258369/taro-demo-test)
**小程序基础库: 无**
**使用框架: React**

### 复现步骤
yarn dev:h5

### 期望结果
hello world

### 实际结果
ReferenceError IS_RN is not defined

### 环境信息
```
👽 Taro v3.5.3


  Taro CLI 3.5.3 environment info:
    System:
      OS: Windows 10
    Binaries:
      Node: 16.14.0 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.19 - C:\Program Files\nodejs\yarn.CMD
      npm: 8.3.1 - C:\Program Files\nodejs\npm.CMD
```

**Comment by TheKonka** [comment]
大哥，```global.d.ts``` 里声明的只是类型，```ts ```不会报错而已，实际上你这个```IS_RN```还是没有声明的啊，可以在```defineConstants``` 里面加下
